src/core/utils/dialog.ts

```diff
@@ -0,0 +1,19 @@
+import { Alert } from 'react-native';
+
+export const confirm = (title: string, message: string): Promise<void> => {
```

Comment:
seeing this in action, made me think how we can make it easier to use. Right now we would always have to handle promise rejection, wich is ugly :)

A better way would be to return a `Promise<boolean>` and always resolve the promise:
- with `True` if used clicked OK
- with `False` if user clicked Cancel

This way we could use this function like this:
```ts
if (await confirm('Title', 'Message')) {...}
```
we can do an exception and not use try..catch in this case, we know that `confirm` will always resolve values.